plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx

```diff
@@ -174,6 +207,8 @@ const BodyWrapper = styled(AccordionItemPanel)`
 const Title = styled.p`
   margin: 0;
   user-select: none;
+  width: 200px;
```

Comment:
Is this 200px going to work in mobile as well?

Reply:
I think 200px is good for mobile as well. setting the width is necessary to make the paragraph take multiple lines

Reply:
What if they have a smaller phone like an iphone 7 or iphone se? Is 200px going to be wider than their screen?

Reply:
iPhone 7's width is 375px and iPhone SE's width is 320px right?